packages/angular/README.md

```diff
@@ -0,0 +1,19 @@
+# Angular support for Bazel
+
+This package is a replacement for parts of the deprecated @angular/bazel package previously maintained by the Angular team.
+
+Currently, this only provides an Angular CLI Builder, which can execute Bazel when triggered by `ng build`, `ng test`, etc.
+See https://angular.io/guide/cli-builder for more info about Builders.
+
+This builder assumes you have already created Bazel configurations (WORKSPACE and BUILD files).
+There is presently no tooling to generate these automatically that's supported by either Angular team or rules_nodejs maintainers.
+See the `@bazel/create` package for a quickstart to creating a Bazel workspace, or look at examples in rules_nodejs.
+
+To use it, you would
+
+```sh
+$ npm install @bazel/angular
+# Edit your angular.json file to use it
+# See examples in this repo
+$ ng build
+```
```

Comment:
nit: add a bit of docs on what goes in the angular.json file

Reply:
done